src/test/java/uk/gov/justice/hmpps/prison/repository/AgencyRepositoryTest.java

```diff
@@ -122,6 +123,14 @@ public void testGetAgencyByType() {
                 .contains("INST", "INST", "INST", "INST", "INST", "INST", "INST", "INST");
     }
 
+    @Test
+    public void testGetAgencies() {
+        final var agencies = repository.getAgencies("agencyId", Order.ASC, 0, 10);
+        assertThat(agencies.getItems()).extracting("agencyId").containsExactly("ABDRCT", "BMI", "BXI", "COURT1", "LEI", "MDI", "MUL", "RNI", "SYI", "TRO");
```

Comment:
normally better to extract by the getter reference so that refactors will pick up automatically